src/transformers/default-schema-transformer.ts

```diff
@@ -129,6 +130,29 @@ function transformByType(question: FormField) {
         ? 'date'
         : question.questionOptions.rendering;
       break;
+    case 'personAttribute':
+      handlePersonAttributeType(question);
+      break;
+  }
+}
+
+async function handlePersonAttributeType(question: FormField) {
+  if (question.questionOptions.rendering !== 'text') {
+    question.questionOptions.rendering === 'ui-select-extended';
+  }
+  const inputFormat = await getPersonAttributeTypeFormat(question.questionOptions?.attributeType);
+  const obj = inputFormat;
+  if (obj?.format === 'org.openmrs.Location') {
+    question.questionOptions.datasource = {
+      name: 'person_attribute_location_datasource',
+    };
+  } else {
```

Comment:
What happens if the configured format is not a concept? Maybe a "org.openmrs.util.AttributableDate"?

Reply:
> What happens if the configured format is not a concept? Maybe a "org.openmrs.util.AttributableDate"?

thanks @samuelmale 
my assumption on this is If the configured format is not `org.openmrs.Location` , the function will still apply the default datasource configuration `(select_concept_answers_datasource)`

Reply:
Don't we want to log some kind of error message?

Reply:
Thanks ,,,i added an error message